src/main/java/org/jabref/Globals.java

```diff
@@ -19,32 +21,26 @@
     public static final RemoteListenerServerLifecycle REMOTE_LISTENER = new RemoteListenerServerLifecycle();
 
     public static final ImportFormatReader IMPORT_FORMAT_READER = new ImportFormatReader();
-
-
+    public static final TaskExecutor taskExecutor = new DefaultTaskExecutor();
```

Comment:
`Globals` is a class I would like to see only go down in size, but I see that the thread pool has to be managed somewhere. I would suggest to add a method here that shuts down everything, the GUI-independent thread pools and this GUI-dependent thread pool and to call this method form `JabRefMain` instead of the current code.